rfc021-platform-support-policy.md

```diff
@@ -65,12 +62,15 @@ Tier 2 supported platforms are those on which Omnitruck will serve packages, but
 * AIX 5.1L
 * FreeBSD 8
 * NetBSD
+* OpenBSD (i386 and amd64)
```

Comment:
If we're working towards having build pipelines for openbsd then it should be tier 1 IMO.  I think the doc should reflect where we want to get to, not necessarily exactly where we are, which means that plans that are currently in-flight should be included.

We do have a long-standing issue with the fact that we're the gateway to teir 1 status and nobody can contribute external packages.  IMO I don't think we should bikeshed on that problem here.  So if we're not delivering the support for OpenBSD then I think it has to stay off of tier 1.  I would love to see that change, but its currently our harsh reality.